R/noteable/4.3.0/Dockerfile

```diff
@@ -1,9 +1,8 @@
-# syntax = docker/dockerfile:1.2.1
+# syntax = docker/dockerfile:1.4
 # Noteable build: adds packages to enable Noteable-specific functionality:
 # - DEX support (via .Rprofile)
-ARG BASE_IMAGE
 # hadolint ignore=DL3006
-FROM ${BASE_IMAGE} as base
+FROM base
```

Comment:
The user should actually get created with id and group 4004:4004; same applies to the python

Reply:
this should be happening in the `base` builds already https://github.com/noteable-io/kernels/blob/9b68bb751feec346454ac42bc57a3b51d504aa64/R/base/4.3.0/Dockerfile#L13-L18 https://github.com/noteable-io/kernels/blob/9b68bb751feec346454ac42bc57a3b51d504aa64/python/base/3.9/Dockerfile#L13-L18